app/models/organization-team-invitation.ts

```diff
@@ -1,18 +1,19 @@
-import Model, { AsyncBelongsTo, attr, belongsTo } from '@ember-data/model';
-import OrganizationModel from './organization';
+import OrganizationInvitationModel from './organization-invitation';
```

Comment:
This is a bad idea. Even tho the two models look similar it better to keep them separate... As in terms of business logic it is different.

Inheritance is model is bad in general case... unless you are inheriting from abstract class. Lets avoid these

Reply:
Duplicate the code along with tests.